readme/markdown.md

```diff
@@ -121,9 +121,15 @@ Note that Mermaid graphs are always rendered on a white background regardless of
 Checkboxes can be added like so:
 
 	- [ ] Milk
-	- [ ] Rice
+	- [x] Rice
 	- [ ] Eggs
 
+Which would turn into:
+
+- [ ] Milk
+- [x] Rice
+- [ ] Eggs
+
```

Comment:
This probably won't work, since the `.md` files are translated into HTML for the web site.

Reply:
Well, it depends on what markdown flavor you use for static website generator. I assumed it is Github flavored. If not, I should either:
1. change it to a photo
2. remove this specific commit

Decision is yours as I don't know what do you use in the backend to generate static website.

Reply:
AFAIK Laurent wrote the generator or the tool that creates the html.

I've just tested your PR and that's how it turns out:

<img width="238" alt="image" src="https://user-images.githubusercontent.com/223439/88464520-91317380-ce89-11ea-88fd-7381bf584f38.png">

Either use an image and add it to `docs/images` or remove that section.

Reply:
and then reference it with `https://joplinapp.org/images/YOUR-IMG-NAME.jpg`